test/Microsoft.DocAsCode.Build.Engine.Tests/DocumentBuilderTest.cs

```diff
@@ -866,7 +911,8 @@ private void BuildDocument(FileCollection files, Dictionary<string, object> meta
                     TemplateManager = new TemplateManager(null, null, new List<string> { _templateFolder }, null, null),
                     TemplateDir = templateFolder,
                     VersionDir = versionDir,
-                    XRefMaps = ImmutableArray.Create("TestData/xrefmap.yml")
+                    XRefMaps = ImmutableArray.Create("TestData/xrefmap.yml"),
+                    XRefServiceUrls = ImmutableArray.Create("http://restfulapiwebservice0627.azurewebsites.net/uids")
```

Comment:
In general UT should not have dependency to remote services, otherwise UT not only relies on network availability but also relies on the stability of remote service. Try use moq to mock a remote service in UT , example http://dotnetliberty.com/index.php/2016/02/22/moq-on-net-core/